python/plotting/configs/samples_run2_2016.py

```diff
@@ -114,27 +118,27 @@ def get_jetbin(channel, category, weight):
-		mediummass = "((genbosonmass >= 50.0 && genbosonmass < 150.0 && (npartons == 0 || npartons >= 5))*1.17315803668195e-4) + ((genbosonmass >= 50.0 && genbosonmass < 150.0 && npartons == 1)*1.621414441741e-5) + ((genbosonmass >= 50.0 && genbosonmass < 150.0 && npartons == 2)*1.6643877999447e-5) + ((genbosonmass >= 50.0 && genbosonmass < 150.0 && npartons == 3)*1.7249743875469e-5) + ((genbosonmass >= 50.0 && genbosonmass < 150.0 && npartons == 4)*1.3442049896748e-5)+"
+		highmass = "((genbosonmass >= 150.0 && (npartons == 0 || npartons >= 5))*3.95423374e-5) + ((genbosonmass >= 150.0 && npartons == 1)*1.27486147e-5) + ((genbosonmass >= 150.0 && npartons == 2)*1.3012785e-5) + ((genbosonmass >= 150.0 && npartons == 3)*1.33802133e-5) + ((genbosonmass >= 150.0 && npartons == 4)*1.09698723e-5)+"
+		mediummass = "((genbosonmass >= 50.0 && genbosonmass < 150.0 && (npartons == 0 || npartons >= 5))*3.95423374e-5) + ((genbosonmass >= 50.0 && genbosonmass < 150.0 && npartons == 1)*1.27486147e-5) + ((genbosonmass >= 50.0 && genbosonmass < 150.0 && npartons == 2)*1.3012785e-5) + ((genbosonmass >= 50.0 && genbosonmass < 150.0 && npartons == 3)*1.33802133e-5) + ((genbosonmass >= 50.0 && genbosonmass < 150.0 && npartons == 4)*1.09698723e-5)+"
 		lowmass = "((genbosonmass < 50.0)*numberGeneratedEventsWeight*crossSectionPerEventWeight)"
 		normalization = "/(numberGeneratedEventsWeight*crossSectionPerEventWeight*sampleStitchingWeight)"
 		return "("+highmass+mediummass+lowmass+")"+normalization
 
 	# DYJetsToLLM_150 sample currently only contains Z->tautau decays
 	def zll_stitchingweight(self):
-		mediummass = "((genbosonmass >= 50.0 && (npartons == 0 || npartons >= 5))*1.17315803668195e-4) + ((genbosonmass >= 50.0 && npartons == 1)*1.621414441741e-5) + ((genbosonmass >= 50.0 && npartons == 2)*1.6643877999447e-5) + ((genbosonmass >= 50.0 && npartons == 3)*1.7249743875469e-5) + ((genbosonmass >= 50.0 && npartons == 4)*1.3442049896748e-5)+"
+		mediummass = "((genbosonmass >= 50.0 && (npartons == 0 || npartons >= 5))*3.95423374e-5) + ((genbosonmass >= 50.0 && npartons == 1)*1.27486147e-5) + ((genbosonmass >= 50.0 && npartons == 2)*1.3012785e-5) + ((genbosonmass >= 50.0 && npartons == 3)*1.33802133e-5) + ((genbosonmass >= 50.0 && npartons == 4)*1.09698723e-5)+"
 		lowmass = "((genbosonmass < 50.0)*numberGeneratedEventsWeight*crossSectionPerEventWeight)"
 		normalization = "/(numberGeneratedEventsWeight*crossSectionPerEventWeight*sampleStitchingWeight)"
 		return "("+mediummass+lowmass+")"+normalization
 
 	def wj_stitchingweight(self):
 		return "(((npartons == 0 || npartons >= 5)*7.09390278348407e-4) + ((npartons == 1)*1.90063898596475e-4) + ((npartons == 2)*5.8529964471165e-5) + ((npartons == 3)*1.9206444928444e-5) + ((npartons == 4)*1.923548021385e-5))/(numberGeneratedEventsWeight*crossSectionPerEventWeight*sampleStitchingWeight)"
-	
+
 	def hadronic_scale_factor(self, channel):
 		if channel in ["mt", "et"]:
-			return "(0.95)"
+			return "(0.97)"
```

Comment:
In fact, for mt and et we use Medium WP in MSSM (in SM it's the Tight) and for tt the Tight WP in MSSM (in SM it'2 the VTight). So I'll move it to the cutstrings, since the cut_type structure is there already settled.

By the way, the function hadronic_scale_factor() in samples_run2_2016.py is applied to ztt only? Is this the right way? As far as I understand this factor is nothing else that an efficiency specific for tau selection, so it does not depend on the process itself. And for tt it should be applied twice, right?